tests/utils/utils.go

```diff
@@ -234,7 +234,77 @@ func SendERC20Destination(
 	Expect(event.Sender).Should(Equal(crypto.PubkeyToAddress(senderKey.PublicKey)))
 	Expect(event.Amount).Should(Equal(bridgedAmount))
 
-	return receipt, event.TeleporterMessageID, event.Amount
+	return receipt, event.Amount
+}
+
+func SendERC20MultihopAndVerify(
+	ctx context.Context,
+	network interfaces.Network,
+	fundedKey *ecdsa.PrivateKey,
+	recipientKey *ecdsa.PrivateKey,
+	recipientAddress common.Address,
+	fromSubnet interfaces.SubnetTestInfo,
+	fromBridge *erc20destination.ERC20Destination,
+	fromBridgeAddress common.Address,
+	toSubnet interfaces.SubnetTestInfo,
+	toBridge *erc20destination.ERC20Destination,
+	toBridgeAddress common.Address,
+	cChainInfo interfaces.SubnetTestInfo,
+	bridgedAmount *big.Int,
```

Comment:
This is a lot of parameters, is there any way we can reduce/combine them? 

For instance, I think we don't necessarily need both `fromBridge` and `fromBridgeAddress`